test/regress/regress0/parser/shadow_fun_symbol_all.smt2

```diff
@@ -1,5 +1,5 @@
 ; EXPECT: Symbol `sin' is shadowing a theory function symbol
-; SCRUBBER: grep -o "Symbol `sin' is shadowing a theory function symbol"
+; SCRUBBER: grep -o "Symbol \`sin' is shadowing a theory function symbol"
```

Comment:
This is because we added `shell=True`. Note that the backtick needs to be escaped when executed in a regular shell as well. I'd thus suggest to leave it like it is right now (in the PR).

Reply:
I see. Thanks for the explanation.